projects/go-style-guide/src/app/features/ui-kit/components/form-docs/components/input-docs/input-docs.component.html

```diff
@@ -351,4 +351,52 @@ <h4 class="go-heading-4 go-heading--underlined">Code</h4>
 
     </div>
   </go-card>
+
+  <go-card id="form-length-type" class="go-column go-column--100">
+    <ng-container go-card-header>
+      <h2 class="go-heading-2 go-heading--no-wrap">Component Maxlength And Minlength</h2>
+      <go-copy cardId="form-length-type" goCopyCardLink></go-copy>
+    </ng-container>
+    <div class="go-container" go-card-content>
+
+      <div class="go-column go-column--100">
+        <p class="go-body-copy">
+          The maxlength and minlength properties of the input can be set via
+          <code class="code-block--inline">@Input() maxlength: number;</code>
+          and the <code class="code-block--inline">@Input() minlength: number;</code> bindings.
+          By default the <code class="code-block--inline">maxlength</code> is set to 
+          <code class="code-block--inline">524288</code> and the 
+          <code class="code-block--inline">minlength</code> is set to <code class="code-block--inline">0</code>.
+          Please note that the <code class="code-block--inline">minlength</code> property would come into
+          effect when the corresponding form containing the input component would be submitted.
+        </p>
+        <p class="go-body-copy go-body-copy--no-margin">
+          In the example below we show how to use <code class="code-block--inline">maxlength</code>
+          and <code class="code-block--inline">minlength</code>.
+        </p>
```

Comment:
Just a thought, but I feel like this paragraph is pretty unnecessary. I know we say something similar in the Component Type card above, but there we are clarifying that specific type of input type is being demonstrated, and so there is a point to the paragraph beyond just essentially saying "here's an example".

Reply:
Removing it right away!